usecases.md

```diff
@@ -3,3 +3,418 @@ layout: base
+
+**Requirements** :
+
+- All requirements as listed by Use Case 1.A, 1.B, or 1.C, as appropriate.
+- User can upload an event forecast, e.g., a time series of true/false values, or a time series of event occurrence likelihood.
+- User can upload corresponding event data or connect to a saved event time series (2.B).
+- User can select among event forecast performance metrics (3.A).
+- Framework provides metric values and visuals of event forecast performance (3.A).
+- Framework provides a report for user download.
+
+### 1.F. Conduct forecast trial
+
+**Use case narrative** : The framework administrator, in consultation with trial participants, defines trial period, locations, forecast quantities, time horizons, time resolutions, etc. of a forecast trial. The trial may be either retrospective or live, and may involve a test period followed by an evaluation period.
+
+**Requirements** :
+
+- All requirements as listed for Use Case 1.C
+- Administrator, in consultation with trial participants, defines data sources, forecast run intervals/horizons, trial period, metrics, filters
+- Framework concatenates regularly posted forecast runs into a continuous forecast.
+- Framework accepts and allows for download of streaming data.
+- Framework generates intermediate reports during the trial.
+
+### 1.G. Compare multiple forecast runs to measurements (stretch)
+
+**Use case narrative** : A framework user uploads a sequence of forecast runs at a regular interval, uploads corresponding data or selects reference data, specifies filters, specifies temporal averaging, and selects metrics. The framework calculates metrics, provides a visual display of forecast performance and an evaluation report for download.
+
+**Requirements** :
+
+- All requirements as listed for Use Case 1.A, for each forecast run.
+- Each forecast run must be identically specified.
+- Forecast runs are issued at a regular interval (e.g., every 6 hours).
+- User can upload multiple forecast runs. Each forecast run must be:
+  - A time series of uniform forecast intervals, duration, and value type (e.g. 5 minute interval, interval average forecasts through 24 hours from the start time)
+- Issued at a regular interval (e.g. every 6 hours).
+- User can define forecast evaluation intervals, horizons, and temporal averaging.
+
+
+
+## 2. Analyze forecasts
+
+### 2.A. Select subsets of forecasts and data
+
+**Use case narrative** : A framework user selects a subset of an uploaded forecast, and the corresponding data, based on one or more conditions, for detailed analysis.  The framework calculates metrics and provides a visual display of forecast performance.
+
+**Requirements** :
+
+- User can select a subset of points in a forecast or its corresponding data based on one or more conditions: forecast error, forecast horizon, time of day, weather (irradiance, temperature, wind). Framework returns selected subsets from the forecast and corresponding data.
```

Comment:
Aidan: how is forecast error defined? Based on any chosen metric?

Reply:
I think in this context, forecast error is the difference between forecast and measured value at a point in time.  It's not a metric like RMSE.

Reply:
Oh, I see. I can revert my change here.

Reply:
We could describe it as the "forecast error (bias)". A stretch goal could be implement selection for other metrics.